client/src/App.js

```diff
@@ -1,17 +1,19 @@
 // @flow
 
-import $ from 'jquery'
```

Comment:
jquery was removed inside of react. If it is needed for some reason (likely working with materialize then access it through `window`). Ideally this is avoided.